{
    "type": "bundle",
    "id": "bundle--a463ceb3-ac7f-463f-bcda-5ac70cc2a1ec",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--4b1f709e-6b9b-46d6-b708-807cd029d1f4",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-01-15T01:42:20.344463Z",
            "modified": "2025-01-15T01:42:20.344463Z",
            "name": "CVE-2024-56374",
            "description": "An issue was discovered in Django 5.1 before 5.1.5, 5.0 before 5.0.11, and 4.2 before 4.2.18. Lack of upper-bound limit enforcement in strings passed when performing IPv6 validation could lead to a potential denial-of-service attack. The undocumented and private functions clean_ipv6_address and is_valid_ipv6_address are vulnerable, as is the django.forms.GenericIPAddressField form field. (The django.db.models.GenericIPAddressField model field is not affected.)",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-56374"
                }
            ]
        }
    ]
}